certificate/certificate.go

```diff
@@ -177,6 +178,25 @@ var ExtKeyUsage = [...]string{
 	"ExtKeyUsageOCSPSigning",
 	"ExtKeyUsageMicrosoftServerGatedCrypto",
 	"ExtKeyUsageNetscapeServerGatedCrypto",
+	"ExtKeyUsageMicrosoftCommercialCodeSigning",
```

Comment:
Noticed that [x509 package](https://golang.org/src/crypto/x509/x509.go?s=20488:24535#L576) had these as further known EKUs.